app/views/hyrax/my/_collection_action_menu.html.erb

```diff
@@ -30,14 +31,15 @@
         <% end %>
       </li>
     <% end %>
-    <%# TODO: Only managers and depositors can add collections to collections. %>
-    <li role="menuitem" tabindex="-1">
-      <%= link_to '#',
-        class: 'itemicon add-collection-to-collection',
-        title: t("hyrax.dashboard.my.action.add_collection_to_collection"),
-        data: { nestable: collection_presenter.collection_type_is_nestable? } do %>
-        <%= t("hyrax.dashboard.my.action.add_collection_to_collection") %>
-      <% end %>
-    </li>
+    <% if current_ability.can_deposit_into_collection?(collection) %>
+      <li role="menuitem" tabindex="-1">
```

Comment:
I see that works index does a similar check, so this should be fine.

Reply:
With the refactor, this should be...  `<% if can? :deposit, collection %>`

I'm not sure the syntax is exactly right, but generally in views, you don't have to reference current_ability for can? queries.

Reply:
With issue #1707 changes, the check here is more complex.  See the issue for details.